<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profile</class>
 <widget class="QWidget" name="profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">profile
{
	background-color: rgb(0, 0, 0);
}
QPushButton
{
	color: rgb(237, 212, 0);
	background-color: rgb(0, 0, 0);
	border-radius:16px;
	border:2px solid yellow;
}
QPushButton:hover
{
	color: rgb(238, 238, 236);
	background-color: rgb(237, 212, 0);
}
QPushButton#btn_logout
{
	color: rgb(164, 0, 0);
	background-color: rgb(0, 0, 0);
	border-radius:16px;
	border:2px solid red;
}
QPushButton:hover#btn_logout
{
	color: rgb(238, 238, 236);
	background-color: rgb(164, 0, 0);
}
QPushButton#btn_back
{
	color: rgb(164, 0, 0);
	background-color: rgb(0, 0, 0);
	border-radius:16px;
	border:2px solid red;
}
QPushButton:hover#btn_back
{
	color: rgb(238, 238, 236);
	background-color: rgb(164, 0, 0);
}
QPlainTextEdit
{
	background-color: rgb(223, 225, 106);
}
QLineEdit
{
	background-color: rgb(237, 212, 0);
}
QLineEdit:hover
{
	background-color: rgb(252, 233, 79);
}</string>
  </property>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>501</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="1" column="2">
     <widget class="QPlainTextEdit" name="show_reserved">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>520</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>520</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="show_sans">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>520</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>520</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="show_movies">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>520</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>520</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_movies">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(237, 212, 0);</string>
      </property>
      <property name="text">
       <string>Movies' Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_sans">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(237, 212, 0);</string>
      </property>
      <property name="text">
       <string>Sans</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_reserve">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(237, 212, 0);</string>
      </property>
      <property name="text">
       <string>Reserved</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_logout">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_back">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>320</y>
     <width>154</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>aakar</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(237, 212, 0);</string>
        </property>
        <property name="text">
         <string>Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineedit_unreserved">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_unreseved">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>UnReseved</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_unresevedall">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>UnReseved All</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
